docs/adguard-mini-for-mac/features/about.md

```diff
@@ -0,0 +1,24 @@
+---
+title: About
+sidebar_position: 7
+---
+
+![About tab](https://cdn.adtidy.org/content/kb/ad_blocker/mini_mac/about.png)
+
+On the *About* tab, you can check:
+
+- Current version of your AdGuard Mini app
```

Comment:
it's an extension

Reply:
I think you can say it's kinda both because you install it from the App Store like any other app, and it has menu bar popup and an app window like a proper app